lightwood/helpers/general.py

```diff
@@ -107,8 +109,8 @@ def evaluate_multilabel_accuracy(true_values, predictions, **kwargs):
 
 
 def evaluate_array_accuracy(
-        true_values: List[List[Union[int, float]]],
-        predictions: List[List[Union[int, float]]],
+        true_values: pd.DataFrame,
```

Comment:
is this always a pandas structure or is Iterable[float etc.] ok?

Reply:
Changed so that it's always pandas, but good catch -- it should be `pd.Series`, not `pd.DataFrame`